packages/app/test/cypress/integration/20-basic-features/use-tools.spec.ts

```diff
@@ -306,18 +306,23 @@ context('Tag Oprations', () =>{
       cy.get('.list-group-item').each(($row) => {
         if($row.find('a').text() === oldPageName){
           cy.wrap($row).within(() => {
-            cy.getByTestid('open-page-item-control-btn').click();
+            cy.getByTestid('open-page-item-control-btn').first().click();
+            cy.getByTestid('page-item-control-menu').should('have.class', 'show').first().within(() => {
+            // eslint-disable-next-line cypress/no-unnecessary-waiting
+            cy.wait(1000);
+            cy.screenshot(`${ssPrefix}2-open-page-item-control-menu`);
+            cy.getByTestid('open-page-move-rename-modal-btn').click({force: true});
+            })
           });
         }
       });
     });
-    cy.screenshot(`${ssPrefix}2-click-three-dots-menu`, {capture: 'viewport'});
 
     cy.getByTestid('search-result-list').within(() => {
       cy.get('.list-group-item').each(($row) => {
         if($row.find('a').text() === oldPageName){
           cy.wrap($row).within(() => {
-            cy.getByTestid('open-page-move-rename-modal-btn').click();
+            cy.getByTestid('open-page-move-rename-modal-btn').click({force: true});
```

Comment:
以下のエラーが出るため　`{force: true}` optionを追加 

````
  1) Tag Oprations
       Successfully rename page from generated tag:
     CypressError: Timed out retrying after 4050ms: `cy.click()` failed because this element is not visible:

`<button type="button" data-testid="open-page-move-rename-modal-btn" tabindex="0" role="menuitem" class="grw-page-control-dropdown-item dropdown-item">...</button>`

This element `<button.grw-page-control-dropdown-item.dropdown-item>` is not visible because its parent `<div.dropdown-menu.dropdown-menu-right>` has CSS property: `display: none`

Fix this problem, or use `{force: true}` to disable error checking.

https://on.cypress.io/element-cannot-be-interacted-with

```